Use provenance url instead domain for sources

shifucun opened the pull request:
Still shows domain in both place page and timeline page.
This bundles the mixer change that removes provenance domain as well. So client and api changes are within one commit.

![image](https://user-images.githubusercontent.com/5951856/107683003-eaaa9580-6c55-11eb-9920-28cebbb4412b.png)
![image](https://user-images.githubusercontent.com/5951856/107709642-64ed1100-6c7a-11eb-966f-e29bfeb2ac2b.png)

beets commented:
there could be multiple sources per chart in the timeline tool, so would be good to display only the domain there as well.
this pr would be a case where the mixer + website change should be in the same pr, so that they can be synced? but i'll approve the mixer pr, so that the mixer submodule in this repo can be brought up to date with the non-backwards compatible change.

pradh commented:
> there could be multiple sources per chart in the timeline tool, so would be good to display only the domain there as well.
> this pr would be a case where the mixer + website change should be in the same pr, so that they can be synced? but i'll approve the mixer pr, so that the mixer submodule in this repo can be brought up to date with the non-backwards compatible change.

nice catch!

shifucun commented:
> there could be multiple sources per chart in the timeline tool, so would be good to display only the domain there as well.
> this pr would be a case where the mixer + website change should be in the same pr, so that they can be synced? but i'll approve the mixer pr, so that the mixer submodule in this repo can be brought up to date with the non-backwards compatible change.

changed to use domain and updated mixer submodule in this PR too.

beets reviewed (approved):
thx!
